### PR TITLE
Zib-AbilityToDrink

### PR DESCRIPTION
**Pull request by Ashwin-nictiz**

### CODE REVIEW

**Review by jduwel — APPROVED** [review]
Terminology gerefresht, verder geen opmerkingen!

File: known-issues.yml [hunk 1]
```diff
@@ -15,6 +15,12 @@ pattern-ZibHealthProfessionalReference:
     StructureDefinition:
       - message: "sd-pg-08: 'The title of the StructureDefinition should conform to the profiling guidelines'"
         reason: The title is shown in the hosting profiles as the name of the datatype. "zib HealthProfessional Reference" is more descriptive than the title according to the profiling guidelines ("pattern ZibHealthProfessionalReference").
+        
+zib-AbilityToDrink:
+  zib deviations:
+   Observation.component:drinkingLimitations.value[x]:
+      - cardinality: 0..1
+        reason: The cardinality mismatch between the zib (0..*) and FHIR (0..1) is explained by the parent .component element that may exists 0..* times, thereby this mapping is still honoring the cardinality requirements of the zib.
```
Comment (LuudSlagter) [inline known-issues.yml:23]: Replace 'may exists' by 'may exist'.

File: resources/zib/zib-AbilityToDrink.xml [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+        <coding>
+          <system value="http://snomed.info/sct" />
+          <code value="288851008" />
+        </coding>
+      </patternCodeableConcept>
+    </element>
+    <element id="Observation.component:drinkingLimitations.value[x]">
+      <path value="Observation.component.value[x]" />
+      <short value="DrinkingLimitations" />
+      <definition value="DrinkingConstraints specifies the patient’s constraints in drinking." />
```
Comment (LuudSlagter) [inline resources/zib/zib-AbilityToDrink.xml:106]: Replace 'DrinkingConstraints' by 'DrinkingLimitations' (and for consistency also 'constraints' by 'limitations'). I'll create a zib ticket to change this in the zib itself as well ('DrinkingConstraints' was the name of this element in the 2015/2016 version).

File: resources/zib/terminology/DrinkBeperkingenCodelijst-2.16.840.1.113883.2.4.3.11.60.40.2.4.8.2--20200901000000.xml [hunk 3]
```diff
@@ -0,0 +1,118 @@
+<ValueSet xmlns="http://hl7.org/fhir">
+   <id value="2.16.840.1.113883.2.4.3.11.60.40.2.4.8.2--20200901000000"/>
+   <meta>
+      <profile value="http://hl7.org/fhir/StructureDefinition/shareablevalueset"/>
+      <!--<profile value="http://hl7.org/fhir/4.0/StructureDefinition/ValueSet"/>-->
```
Comment (LuudSlagter) [inline resources/zib/terminology/DrinkBeperkingenCodelijst-2.16.840.1.113883.2.4.3.11.60.40.2.4.8.2--20200901000000.xml:5]: Can this line be removed (general question for ValueSets)?

File: resources/zib/zib-AbilityToDrink.xml (hunk 2 above)
Comment (jduwel) [reply resources/zib/zib-AbilityToDrink.xml:106]: We add this stuff automatically from the zib definition, so we don't manually edit these in our profiles. This is the responsibility of the zib-centrum, so creating a zib ticket to have this corrected is the way to go (although there are so much of these kinds of errors that I wonder when you'll get tired of it).

**Comment by LuudSlagter** [comment]
There still is a validation error regarding a code that apparently isn't present in DrinkenCodelijst; however the code in question is part of that ValueSet.

**Comment by LuudSlagter** [comment]
Validation errors w.r.t. examples are not valid, as the code is present in the Dutch extension of SNOMED (checked by performing a local QA).

**Pull request merged**